docs/registering-ticket-sales-multiple.md

```diff
@@ -0,0 +1,158 @@
+# Registering multiple ticket sales at once
+
+This mini-guide illustrates how to register UiTPAS discounted ticket sales for multiple tickets at once. It builds upon the [Registering ticket sales guide](/docs/registering-ticket-sales-group.md) which you need to read first to learn more about authentication, the work flow and registering events.
```

Comment:
Same suggestion about "mini-guide"

Reply:
Also "work flow" should be "workflow" as mentioned in the other comment on the other guide